SerialPrograms/Source/PokemonSV/Programs/TeraRaids/PokemonSV_TeraSelfFarmer.cpp

```diff
@@ -277,6 +278,14 @@ void TeraSelfFarmer::program(SingleSwitchProgramEnvironment& env, BotBaseContext
 //            env.log("Detected a " + TERA_TYPE_NAMES[(size_t)type] + " " + std::to_string(stars) + " star raid.", COLOR_PURPLE);
         }
 
+        if (MODE == Mode::FARM_LP_ONLY) {

```

Comment:
I'd say to do this : 

bool skip = false;
if (MODE == Mode::FARM_LP_ONLY){
    skip = true;
}
if (stars < MIN_STARS || stars > MAX_STARS){
    skip = true;
}

if (skip(){
    skip
}

Reply:
Done.